recipe/meta.yaml

```diff
@@ -1,27 +1,27 @@
 {% set name = "ruptures" %}
-{% set version = "1.1.2" %}
+{% set version = "1.1.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: fb6eda550d749d63d0ea1582f37d290fffa5fd6dc61513d971ef3429681d5255
+  sha256: e3845f5ccc91fe958a120b6bfbc35092b690f09e76d16df4b550b6362ac67b67
 
 build:
-  number: 2
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-build-isolation -vv
```

Comment:
Have you tried without the `--no-build-isolation` option ? 
I added it in the first place because I initially thought this could help it the error was related to a isolation problem, but now that specifying `setuptools_scm` fixes the issues, it might be okay to remove it.

Reply:
Remved in cf7ab97

Let us see what the CI tells us.

Reply:
it works!